tcs-api/src/main/java/com/transformuk/hee/tis/tcs/api/dto/ProgrammeMembershipSummaryDTO.java

```diff
@@ -0,0 +1,15 @@
+package com.transformuk.hee.tis.tcs.api.dto;
+
+import java.io.Serializable;
+import java.time.LocalDate;
+import lombok.Data;
+
+/**
+ * A DTO for the ProgrammeMembershipSummary entity.
+ */
+@Data
+public class ProgrammeMembershipSummaryDTO implements Serializable {
+  private String programmeMembershipId;
+  private String programmeMembershipName;
```

Comment:
Do we mean 'programmeName' here? Otherwise I'm not sure what this field would contain, so it should have a comment if its not a std. ProgrammeMembership field I think.